postprocess/non-openshift.jsonnet

```diff
@@ -0,0 +1,91 @@
+local com = import 'lib/commodore.libjsonnet';
+
+local inv = com.inventory();
+local distribution = inv.parameters.facts.distribution;
+
+local chart_output_dir = std.extVar('output_path');
+
+local list_dir(dir, basename=true) =
+  std.native('list_dir')(dir, basename);
+
+local chart_files = list_dir(chart_output_dir);
+local input_file(elem) = chart_output_dir + '/' + elem;
+
+local stem(elem) =
+  local elems = std.split(elem, '.');
+  std.join('.', elems[:std.length(elems) - 1]);
+
+local fixup_deploy(obj) =
+  obj {
+    spec+: {
+      template+: {
+        spec+: {
+          containers: [
+            c {
+              ports: [
+                {
+                  containerPort: 8080,
+                  name: 'http',
+                },
+              ],
+            }
+            for c in super.containers
+            if c.name != 'kube-rbac-proxy'
+          ],
+          volumes: [
+            v
+            for v in super.volumes
+            if v.name != 'tls-cert'
+          ],
+        },
+      },
+    },
+  };
+local fixup_service(obj) =
+  obj {
+    spec+: {
+      ports: [
+        {
+          name: 'http',
+          port: 8080,
+          targetPort: 'http',
+        },
+      ],
+    },
+  };
+local fixup_servicemonitor(obj) =
+  obj {
+    spec+: {
+      endpoints: [
+        {
+          interval: '30s',
+          port: 'http',
+          scheme: 'http',
+        },
+      ],
+    },
+  };
+
+local fixup_obj(obj) =
+  if distribution != 'openshift4' then (
```

Comment:
Couldn't we do that check before we load and parse all files?

Reply:
Because I thought postprocessing would eat the inputs if it has no outputs, but I appear to have changed/fixed that at some point :)

Reply:
updated in latest version